app/views/layouts/application.html.erb

```diff
@@ -28,6 +28,13 @@
         <ul class="nav navbar-nav">
           <%= render partial: 'layouts/nav/group', collection: @nav_groups %>
           <%= nav_link("All Apps", "th", apps_index_path) if Configuration.show_all_apps_link? %>
+          <% if Configuration.app_development_enabled? || @nav_groups.has_batch_connect_apps? %>
+          <li>
+            <a target="_self" href="<%= batch_connect_sessions_path %>">
+              <i class="fa fa-th"></i><span id="interactive-sessions-nav-text" class="hidden-sm hidden-md"> Interactive Sessions</span>
```

Comment:
Instead of `interactive-sessions-nav-text` id, can't you achieve the same thing by just using class="visible-lg-*" or class="hidden-sm hidden-md hidden-lg".

Reply:
But actually instead of visible the hidden classes are better so you can keep the text visible at the smallest width.